extensions/Today's actual date

```diff
@@ -0,0 +1,32 @@
+class DateTimeExtension {
```

Comment:
Extensions that are submitted must be wrapped in an [immediately-invoked-function-expression (IIFE)](https://docs.turbowarp.org/development/extensions/unsandboxed#:~:text=The%20unsandboxed%20version%20would%20have%20code%20like%20this%3A).